apps/prairielearn/assets/scripts/lib/questionsTable.ts

```diff
@@ -48,28 +48,28 @@ onDocumentReady(() => {
   } = decodeData('questions-table-data');
 
   window.topicList = function () {
-    const data = $('#questionsTable').bootstrapTable('getData');
-    return _.keyBy(_.map(data, (row) => row.topic.name));
+    const data = $('#questionsTable').bootstrapTable('getData') as QuestionsPageData[];
+    return _.keyBy(data.map((row) => row.topic.name));
```

Comment:
This produces an object like `{"a": "a", "b": "b"}`. I believe we could pretty easily replace that with `Object.fromEntries`, something like this:

```suggestion
  return Object.fromEntries(data.map(({ topic }) => [topic.name, topic.name]));
```

Ditto in some other places in this file.

Reply:
Done in f1a64ba3f49f8c8d0d1d012255480bbd7f1752a6 and 1719fb73e76a8dd0852f941e723bd023df672649.